.github/workflows/testme.yml

```diff
@@ -1,27 +1,49 @@
 on:
   push:
-  
+
 jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/setup-python@v4
-      with:
-        python-version: '3.10'
-    - name: deps
-      run: pip install ruff
-    - uses: actions/checkout@v3
-    - name: lint
-      run: ruff upload.py
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: deps
+        run: pip install ruff
+      - uses: actions/checkout@v3
+      - name: lint
+        run: ruff upload.py
```

Comment:
My formatter (prettier) did this formatting.

If you want to keep it in the old style, I can revert this

Reply:
no preference

if the computer is happy I'm happy